### PR TITLE
{bio}[GCCcore/10.3.0,GCCcore/11.2.0] paladin v1.4.6

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ('cURL', '7.78.0'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+files_to_copy = [(['paladin'], 'bin'), 'run.py', 'sample_data', 'scripts']
+
+sanity_check_paths = {
+    'files': ['bin/paladin'],
+    'dirs': ['sample_data'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb:39]: @verdurin
can you add a `sanity_check_commands` here? for example
```
sanity_check_commands = ['paladin --help']
```

File: easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ('binutils', '2.37'),
+]
+
+# Use EB build flags
+buildopts = 'CFLAGS="$CFLAGS"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('cURL', '7.78.0'),
+    ('OpenSSL', '1.1', '', True),
```
Comment (smoors) [inline easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb:31]:
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```

File: easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-10.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    ('binutils', '2.36.1'),
+]
+
+# Use EB build flags
+buildopts = 'CFLAGS="$CFLAGS"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('cURL', '7.76.0'),
+    ('OpenSSL', '1.1', '', True),
```
Comment (smoors) [inline easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-10.3.0.eb:31]:
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```

File: easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb:41]: @smoors now added

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node306.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/4bcaeed24084b544aacf0b25fd57f61b for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node306.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/9e0690f7833484dbef6e6c99eeb7e420 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**